src/main/java/com/p6spy/engine/wrapper/ConnectionWrapper.java

```diff
@@ -210,7 +210,17 @@ public String nativeSQL(String sql) throws SQLException {
 
   @Override
   public void setAutoCommit(boolean autoCommit) throws SQLException {
-    delegate.setAutoCommit(autoCommit);
+    SQLException e = null;
+    boolean oldAutoCommit = delegate.getAutoCommit();
+    try {
+      jdbcEventListener.onBeforeSetAutoCommit(connectionInformation,autoCommit,oldAutoCommit);
```

Comment:
Please make it consistent with other source code - spaces between arguments.

Reply:
done